pyaerocom/io/read_mscw_ctm.py

```diff
@@ -302,6 +318,8 @@ def read_var(self, var_name, ts_type=None, **kwargs):
             data.time.attrs['standard_name'] = 'time'
             data.attrs['units'] = self.preprocess_units(data.units, EMEP_prefix)
             cube = data.to_iris()
+            if ts_type == 'hourly':
+                cube.coords()[0].convert_units('hours since 1900-01-01')
```

Comment:
Is it certain that first index is **always** the time dimension?

Reply:
Maybe it is better to try `cube.coord('time')` to make sure you are acting on the time dimension.

Reply:
```suggestion
                cube.coords('time').convert_units('hours since 1900-01-01')
```

Reply:
Good point. Not sure how much the EMEP files can vary, this seems like a better solution.